reviewratings/src/main/java/com/taufiqrahman/reviewratings/Bar.java

```diff
@@ -26,6 +26,7 @@
     private String starLabel;
     private int raters;
     private int color;
+    private int endColor;
```

Comment:
had doubts if I should use `endColor` and `startColor` because `color` wouldn't be that explicit that it's the start color. 
As well, maybe only provide a setter/getter for startColor that would change the `color` property, but didn't want to be too verbose about it. 

What do you think?

Reply:
Keep `startColor` because the code would be easier to comprehend at a glance for anyone.

Provide a setter/getter for `startColor` that would change the color property for the RatingReviews constructor you made. It's enough verbosity as it directly serves the purpose of the code.